app/controllers/application_controller.rb

```diff
@@ -1,8 +1,8 @@
 class ApplicationController < ActionController::API
   include Error::ErrorHandler
 
-  def represent_response raw_response
-    render json: { status: true, data: raw_response.as_json }, status: Settings.http_code.code_201
+  def represent_response raw_response, http_code = Settings.http_code.code_201
```

Comment:
(optional) nên dùng chung 1 interface với cái hàm render 
```suggestion
  def represent_response raw_response, status = Settings.http_code.code_201
```